main.go

```diff
@@ -361,7 +361,7 @@ func main() {
 		SecretDataBuilder:          secretDataBuilder,
 		HMACValidator:              hmacValidator,
 		ClientFactory:              clientFactory,
-		BackOffRegistry:            controllers.NewBackOffRegistry(backOffOpts...),
+		BackOffRegistry:            controllers.NewBackOffRegistry(backoffOpts...),
```

Comment:
```suggestion
		BackoffRegistry:            controllers.NewBackOffRegistry(backoffOpts...),
```

Reply:
Good question. I decided to leave it as-is since the backoff library we are using spells  it 'BackOff'